configure.ac

```diff
@@ -238,7 +238,7 @@ AS_IF([test "x$enable_require_system_font_provider" != xno  dnl
         && test "x$directwrite" != xtrue                    dnl
         && test "x$coretext" != xtrue                       ], [
     AC_MSG_ERROR(m4_text_wrap(m4_normalize([
-            Either DirectWrite (on Windows), CoreText (on OSX), or Fontconfig
+            Either DirectWrite (on Windows), Core Text (Apple), or Fontconfig
             (Linux, other) is required. If you really want to compile without
             a system font provider, add --disable-require-system-font-provider]),
```

Comment:
While we’re at it: this is technically not quite correct or at least misleading: Fontconfig is available on all platforms not just “Linux, other” and providing just Fontconfig also sufficies on Windows and Apple platforms

Reply:
Maybe: “At least one of Core Text (Apple exclusive), DirectWrite (Windows exclusive) or Fontconfig is required.”?

Reply:
Added a commit that switches to (almost) your suggested wording.